### PR TITLE
PersonaCoin component will either render the initials or the avatar image but not both

### PR DESCRIPTION
**Pull request by s-vaibhav**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

In case the user image was unavailable, PersonaCoin was rendering the initials first and then over it a div tag with img tag in it whose src was empty. This was blocking any mouse-move event (bound to the initials) as it was below the img tag and since the img tag was 40px X 0px, mouse move events were not captured there too.

**UPDATE**:
PersonalCoin will render the initials based on the valid conditions and if we'll have a valid image URL and do not wish to hide it, the image will be rendered (over the initials) too. This will ensure that the img tag always comes with a valid src.
This is done as behaviour related to 'showInitialsUntilImageLoads' property was breaking.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
Great. I'm glad you caught this. I was wondering when the feature was initially added if we should watch out for this.

**Comment by micahgodbolt** [comment]
adding @Markionium on really quick as he's the one that initially added the feature.

**Comment by Markionium** [comment]
@micahgodbolt This doesn't have anything to do with the thing that we added a few weeks ago :)

This empty image is also present in Fabric 5. 
https://github.com/OfficeDev/office-ui-fabric-react/blob/5.0/packages/office-ui-fabric-react/src/components/Persona/__snapshots__/Persona.test.tsx.snap#L120-L142

Non the less this is a good thing to remove. It does however relate to the discussion that David started around if the DOM is part of the contract.

**Comment by s-vaibhav** [comment]
Updated the pull request as ‘showInitialsUntilImageLoads’ behaviour was breaking.

File: common/changes/office-ui-fabric-react/master_2018-07-25-11-51.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "VSO: 603037:- Either render the initials or the avtar image (PersonaCoin) and not both in the site permission panel",
```
Comment (mtennoe) [inline common/changes/office-ui-fabric-react/master_2018-07-25-11-51.json]: typo: avtar => avatar

Comment (mtennoe) [inline common/changes/office-ui-fabric-react/master_2018-07-25-11-51.json]: The comment of the change is very specific to the application it is being integrated into. It should be more generic with focus on what it changes in fabric-react instead

**Comment by s-vaibhav** [comment]
Updated the description and the condition too.
PersonaCoin will need to have a valid image URL to render the user's avatar.

**Comment by mtennoe** [comment]
@mikewheaton, @yiminwu - Seems like we need two code owner approvals for some reason, so mine is not sufficient. Can you give it a quick look, and merge when/if you approve?

**Comment by micahgodbolt** [comment]
@s-vaibhav please resolve that conflicts, then I can push this through.

**Comment by dzearing** [comment]
@s-vaibhav can you resolve conflicts?

**Comment by s-vaibhav** [comment]
@dzearing  I was able to resolve the conflict, but since I merged the OfficeDev/master into my local branch the pre-commit hook is failing while giving the following errors:
```
apps/vr-tests/src/stories/Dropdown.stories.tsx:95:28                                                           
ERROR: 95:28   jsx-no-lambda  Lambdas are forbidden in JSX attributes due to their rendering performance impact
ERROR: 103:22  jsx-no-lambda  Lambdas are forbidden in JSX attributes due to their rendering performance impact
ERROR: 121:23  jsx-no-lambda  Lambdas are forbidden in JSX attributes due to their rendering performance impact
ERROR: 137:26  jsx-no-lambda  Lambdas are forbidden in JSX attributes due to their rendering performance impact
```
Hence I am unable to commit to the same branch. Should I close this pull request and create a new one (from a fresh copy of OfficeDev/master) to avoid this failure at my end?
Or will this be rectified soon?

PS: I haven't modified the ```Dropdown.stories.tsx``` file

**Comment by dzearing** [comment]
@s-vaibhav I'm a little confused. If you have no changes in dropdown.stories, I don't think that should happen. Are you sure you merged upstream/master and not origin/master?

**Comment by dzearing** [comment]
I've attempted to resolve for you.

**Comment by s-vaibhav** [comment]
@dzearing Thank you so much for the help!
This change also required the snapshots to be updated, I have pushed the same.
@micahgodbolt Please see if this can be merged and shipped.

**Pull request merged**